app/models/course/condition/assessment.rb

```diff
@@ -0,0 +1,36 @@
+class Course::Condition::Assessment < ActiveRecord::Base
+  include ActiveSupport::NumberHelper
+
+  acts_as_condition
+  belongs_to :assessment, class_name: Course::Assessment.name, inverse_of: false
+
+  default_scope { includes(:assessment) }
+
+  def title
+    if minimum_grade_percentage
+      self.class.human_attribute_name(title.title,
```

Comment:
I think it only calls this I18n [translation](https://github.com/Coursemology/coursemology2/pull/658/files#diff-2e9896fc197652799a0bbc1f4561b18dR5)